platform/tron/api.go

```diff
@@ -82,10 +82,9 @@ func (p *Platform) GetTokenTxsByAddress(address, token string) (blockatlas.TxPag
 
 	var tokenInfo AssetInfo
 	info, err := p.client.GetTokenInfo(token)
-	if err != nil && len(info.Data) == 0 {
+	if err != nil || len(info.Data) == 0 {
```

Comment:
is this intended? it might cause issues

Reply:
yes, the old logic doesn't make a lot of sense